### DISCUSSION TITLE
Publishing betas

### DISCUSSION THREAD

**Discussion by wdstat** [felixmariotto/three-mesh-ui#257] [Q&A]
Hey guys,

I was a bit upset when I saw no betas of three-mesh-ui in npm.

I want to check v7 in my code before the release. It would let newcomers avoid compiling and using smth like symlinks and other stuff.

Would you please enable the publishing of betas to npm? It would let more people get involved in testing, I guess.

PS
Great job, I can't wait to apply the new possibilities of the new version.

**Comment by swingingtom** [comment]
Hi @wdstat, agreed. And this is on me.
Would you be okay to help on that? Thanks

**Reply by wdstat** [reply]
What kind of help do you need?

**Reply by wdstat** [reply]
I could edit the github pipeline to make it possible to publish betas.

**Reply by swingingtom** [reply]
That would indeed already help! Thanks

**Reply by wdstat** [reply]
Great!
When I have more time, I'll send you a PR. Thank you!

**Comment by wdstat** [comment]
Hi @swingingtom,

Is the current branch for v7 the 7.1.1 one? Am I right? I see 7. x.x, but the latest changes are in 7.1.1.
Which branch should I use to create the PR?

**Reply by swingingtom** [reply]
I would like to keep 7.x.x as v7 main branch. I didn't merged 7.1.1 yet

**Reply by brokvolchansky** [reply]
Ok
